File: live_stream_app/lib/modules/sign_up/model/sign_up_request.dart
class SignUpRequest {
  SignUpRequest({
    this.firstName,
    this.lastName,
    this.birthDay,
    this.address,
    this.email,
    this.username,
    this.password,
    this.file,
    this.role,
  });

  SignUpRequest.fromJson(dynamic json) {
    firstName = json['firstName'];
    lastName = json['lastName'];
    birthDay = json['birthDay'];
    address = json['address'];
    email = json['email'];
    username = json['username'];
    password = json['password'];
    file = json['file'];
    role = json['role'] != null ? json['role'].cast<String>() : [];
  }

  String? firstName;
  String? lastName;
  String? birthDay;
  String? address;
  String? email;
  String? username;
  String? password;
  String? file;
  List<String>? role;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['firstName'] = firstName;
    map['lastName'] = lastName;
    map['birthDay'] = birthDay;
    map['address'] = address;
    map['email'] = email;
    map['username'] = username;
    map['password'] = password;
    map['file'] = file;
    map['role'] = role;
    return map;
  }
}

File: spring-boot-refresh-token-jwt-master/src/main/java/com/bezkoder/spring/security/jwt/payload/request/SignupRequest.java
package com.bezkoder.spring.security.jwt.payload.request;

import java.util.Set;

import javax.validation.constraints.*;
 
public class SignupRequest {
    @NotBlank
    @Size(min = 3, max = 222)
    private String firstName;

    @NotBlank
    @Size(min = 3, max = 222)
    private String lastName;

    @NotBlank
    @Size(min = 3, max = 255)
    private String birthDay;

    @NotBlank
    @Size(min = 3, max = 255)
    private String address;

    @NotBlank
    @Size(max = 50)
    @Email
    private String email;

    @NotBlank
    @Size(min = 3, max = 20)
    private String username;
    
    @NotBlank
    @Size(min = 6, max = 40)
    private String password;

    @NotBlank
    @Size(min = 3, max = 255)
    private String file;

    private Set<String> role;
  
    public String getUsername() {
        return username;
    }
 
    public void setUsername(String username) {
        this.username = username;
    }
 
    public String getEmail() {
        return email;
    }
 
    public void setEmail(String email) {
        this.email = email;
    }
 
    public String getPassword() {
        return password;
    }
 
    public void setPassword(String password) {
        this.password = password;
    }
    
    public Set<String> getRole() {
      return this.role;
    }
    
    public void setRole(Set<String> role) {
      this.role = role;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getBirthDay() {
        return birthDay;
    }

    public void setBirthDay(String birthDay) {
        this.birthDay = birthDay;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getFile() {
        return file;
    }

    public void setFile(String file) {
        this.file = file;
    }
}
